articles/fleetctl.md

```diff
@@ -128,7 +128,7 @@ An API-only user does not have access to the Fleet UI. Instead, it's only purpos
 
 ### Create API-only user
 
-Before creating the API-only user, log in to fleetctl as an admin.  See [authentication](https://#authentication) above for details.
+Before creating the API-only user, log in to fleetctl as an admin.  See [authentication](https://#authentication) above for details. <<< This link is broken...
```

Comment:
I think this is broken because it has "https://" in front of the anchor. This should fix it:
```suggestion
Before creating the API-only user, log in to fleetctl as an admin.  See [authentication](#authentication) above for details.
```